<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Topic 3 Primary and Secondary Databases | Genome, transcriptome and structure databases</title>
  <meta name="description" content="Topic 3 Primary and Secondary Databases | Genome, transcriptome and structure databases" />
  <meta name="generator" content="bookdown 0.22 and GitBook 2.6.7" />

  <meta property="og:title" content="Topic 3 Primary and Secondary Databases | Genome, transcriptome and structure databases" />
  <meta property="og:type" content="book" />
  
  
  
  <meta name="github-repo" content="rstudio/bookdown-demo" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Topic 3 Primary and Secondary Databases | Genome, transcriptome and structure databases" />
  
  
  

<meta name="author" content="Dan MacLean" />


<meta name="date" content="2021-07-07" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="motivation.html"/>
<link rel="next" href="genomics-database-and-genome-assembly.html"/>
<script src="libs/header-attrs-2.9/header-attrs.js"></script>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<link href="libs/anchor-sections-1.0.1/anchor-sections.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.0.1/anchor-sections.js"></script>




<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Databases</a></li>

<li class="divider"></li>
<li class="chapter" data-level="1" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i><b>1</b> Setting up</a>
<ul>
<li class="chapter" data-level="1.1" data-path="index.html"><a href="index.html#prerequisites"><i class="fa fa-check"></i><b>1.1</b> Prerequisites</a>
<ul>
<li class="chapter" data-level="1.1.1" data-path="index.html"><a href="index.html#knowledge-prerequisites"><i class="fa fa-check"></i><b>1.1.1</b> Knowledge prerequisites</a></li>
<li class="chapter" data-level="1.1.2" data-path="index.html"><a href="index.html#software-prerequisites"><i class="fa fa-check"></i><b>1.1.2</b> Software prerequisites</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="2" data-path="motivation.html"><a href="motivation.html"><i class="fa fa-check"></i><b>2</b> Motivation</a>
<ul>
<li class="chapter" data-level="2.1" data-path="motivation.html"><a href="motivation.html#three-weeks-of-computational-analysis-can-save-you-an-hour-of-database-searches"><i class="fa fa-check"></i><b>2.1</b> Three weeks of computational analysis can save you an hour of database searches</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="primary-and-secondary-databases.html"><a href="primary-and-secondary-databases.html"><i class="fa fa-check"></i><b>3</b> Primary and Secondary Databases</a>
<ul>
<li class="chapter" data-level="3.1" data-path="primary-and-secondary-databases.html"><a href="primary-and-secondary-databases.html#about-this-chapter"><i class="fa fa-check"></i><b>3.1</b> About this chapter</a></li>
<li class="chapter" data-level="3.2" data-path="primary-and-secondary-databases.html"><a href="primary-and-secondary-databases.html#primary-databases"><i class="fa fa-check"></i><b>3.2</b> Primary databases</a></li>
<li class="chapter" data-level="3.3" data-path="primary-and-secondary-databases.html"><a href="primary-and-secondary-databases.html#examples-of-primary-databases"><i class="fa fa-check"></i><b>3.3</b> Examples of primary databases</a>
<ul>
<li class="chapter" data-level="3.3.1" data-path="primary-and-secondary-databases.html"><a href="primary-and-secondary-databases.html#ena"><i class="fa fa-check"></i><b>3.3.1</b> ENA</a></li>
<li class="chapter" data-level="3.3.2" data-path="primary-and-secondary-databases.html"><a href="primary-and-secondary-databases.html#genbank"><i class="fa fa-check"></i><b>3.3.2</b> GenBank</a></li>
<li class="chapter" data-level="3.3.3" data-path="primary-and-secondary-databases.html"><a href="primary-and-secondary-databases.html#geo"><i class="fa fa-check"></i><b>3.3.3</b> GEO</a></li>
<li class="chapter" data-level="3.3.4" data-path="primary-and-secondary-databases.html"><a href="primary-and-secondary-databases.html#biostudies"><i class="fa fa-check"></i><b>3.3.4</b> BioStudies</a></li>
<li class="chapter" data-level="3.3.5" data-path="primary-and-secondary-databases.html"><a href="primary-and-secondary-databases.html#protein-data-bank"><i class="fa fa-check"></i><b>3.3.5</b> Protein Data Bank</a></li>
</ul></li>
<li class="chapter" data-level="3.4" data-path="primary-and-secondary-databases.html"><a href="primary-and-secondary-databases.html#secondary-databases"><i class="fa fa-check"></i><b>3.4</b> Secondary databases</a></li>
<li class="chapter" data-level="3.5" data-path="primary-and-secondary-databases.html"><a href="primary-and-secondary-databases.html#examples-of-secondary-databases"><i class="fa fa-check"></i><b>3.5</b> Examples of secondary databases</a>
<ul>
<li class="chapter" data-level="3.5.1" data-path="primary-and-secondary-databases.html"><a href="primary-and-secondary-databases.html#ensembl"><i class="fa fa-check"></i><b>3.5.1</b> Ensembl</a></li>
<li class="chapter" data-level="3.5.2" data-path="primary-and-secondary-databases.html"><a href="primary-and-secondary-databases.html#uniprotkb"><i class="fa fa-check"></i><b>3.5.2</b> UniProtKB</a></li>
</ul></li>
<li class="chapter" data-level="3.6" data-path="primary-and-secondary-databases.html"><a href="primary-and-secondary-databases.html#metadata"><i class="fa fa-check"></i><b>3.6</b> Metadata</a>
<ul>
<li class="chapter" data-level="3.6.1" data-path="primary-and-secondary-databases.html"><a href="primary-and-secondary-databases.html#minimum-information-standards"><i class="fa fa-check"></i><b>3.6.1</b> Minimum Information Standards</a></li>
<li class="chapter" data-level="3.6.2" data-path="primary-and-secondary-databases.html"><a href="primary-and-secondary-databases.html#controlled-vocabularies"><i class="fa fa-check"></i><b>3.6.2</b> Controlled Vocabularies</a></li>
<li class="chapter" data-level="3.6.3" data-path="primary-and-secondary-databases.html"><a href="primary-and-secondary-databases.html#conclusion"><i class="fa fa-check"></i><b>3.6.3</b> Conclusion</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="4" data-path="genomics-database-and-genome-assembly.html"><a href="genomics-database-and-genome-assembly.html"><i class="fa fa-check"></i><b>4</b> Genomics Database and Genome Assembly</a>
<ul>
<li class="chapter" data-level="4.1" data-path="genomics-database-and-genome-assembly.html"><a href="genomics-database-and-genome-assembly.html#about-this-chapter-1"><i class="fa fa-check"></i><b>4.1</b> About this chapter</a></li>
<li class="chapter" data-level="4.2" data-path="genomics-database-and-genome-assembly.html"><a href="genomics-database-and-genome-assembly.html#interactive-tutorial"><i class="fa fa-check"></i><b>4.2</b> Interactive tutorial</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="transcriptomics-databases-and-transcript-abundance.html"><a href="transcriptomics-databases-and-transcript-abundance.html"><i class="fa fa-check"></i><b>5</b> Transcriptomics Databases and Transcript Abundance</a>
<ul>
<li class="chapter" data-level="5.1" data-path="transcriptomics-databases-and-transcript-abundance.html"><a href="transcriptomics-databases-and-transcript-abundance.html#about-this-chapter-2"><i class="fa fa-check"></i><b>5.1</b> About this chapter</a></li>
<li class="chapter" data-level="5.2" data-path="transcriptomics-databases-and-transcript-abundance.html"><a href="transcriptomics-databases-and-transcript-abundance.html#interactive-tutorial-1"><i class="fa fa-check"></i><b>5.2</b> Interactive tutorial</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="structural-databases.html"><a href="structural-databases.html"><i class="fa fa-check"></i><b>6</b> Structural Databases</a>
<ul>
<li class="chapter" data-level="6.1" data-path="structural-databases.html"><a href="structural-databases.html#about-this-chapter-3"><i class="fa fa-check"></i><b>6.1</b> About this chapter</a></li>
<li class="chapter" data-level="6.2" data-path="structural-databases.html"><a href="structural-databases.html#interactive-tutorial-2"><i class="fa fa-check"></i><b>6.2</b> Interactive tutorial</a></li>
</ul></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Genome, transcriptome and structure databases</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="primary-and-secondary-databases" class="section level1" number="3">
<h1><span class="header-section-number">Topic 3</span> Primary and Secondary Databases</h1>
<div id="about-this-chapter" class="section level2" number="3.1">
<h2><span class="header-section-number">3.1</span> About this chapter</h2>
<ol style="list-style-type: decimal">
<li>Questions:</li>
</ol>
<ul>
<li>What is the difference between a primary and secondary database?</li>
<li>What are some examples of primary and secondary databases?</li>
<li>How can I interact with them to find what I need?</li>
<li>What is metadata?</li>
</ul>
<ol start="2" style="list-style-type: decimal">
<li>Objectives:</li>
</ol>
<ul>
<li>Understand the role of a primary and secondary database</li>
<li>Know where to access and how to find training resources provided by the databases</li>
<li>Understand the importance of clear structured metadata</li>
</ul>
<ol start="3" style="list-style-type: decimal">
<li>Keypoints:</li>
</ol>
<ul>
<li>Primary databases are repositories of experimental data</li>
<li>Secondary databases are repositories of results based on the primary data</li>
<li>Metadata is what makes data findable and useable</li>
</ul>
</div>
<div id="primary-databases" class="section level2" number="3.2">
<h2><span class="header-section-number">3.2</span> Primary databases</h2>
<p>In many fields, databases can be categorized as either primary or secondary. The main difference is actually quite simple, <em>Primary</em> databases hold data that come from experiments or direct data gathering exercises and their contents are fixed. A primary database is often called an archive or a database of scientific record.</p>
</div>
<div id="examples-of-primary-databases" class="section level2" number="3.3">
<h2><span class="header-section-number">3.3</span> Examples of primary databases</h2>
<div id="ena" class="section level3" number="3.3.1">
<h3><span class="header-section-number">3.3.1</span> ENA</h3>
<p><a href="https://www.ebi.ac.uk/ena/browser/home">ENA - https://www.ebi.ac.uk/ena/browser/home</a></p>
<p>The European Nucleotide Archive (ENA) is a comprehensive record of the world’s nucleotide sequencing information, covering raw sequencing data and sequence assemblies. Access to it is provided through a genome browser, search forms and tools, and an API - an interface that allows you to write programs that extract data. Like many databases, user training is a very important part of the mission of the database, so they provide a very comprehensive manual and how to’s at <a href="https://ena-docs.readthedocs.io/en/latest/">https://ena-docs.readthedocs.io/en/latest/</a></p>
</div>
<div id="genbank" class="section level3" number="3.3.2">
<h3><span class="header-section-number">3.3.2</span> GenBank</h3>
<p><a href="https://www.ncbi.nlm.nih.gov/genbank/">GenBank - https://www.ncbi.nlm.nih.gov/genbank</a></p>
<p>GenBank is another distinct genetic sequence database with annotated sequences and it aims to hold <em>all</em> publicly available DNA sequences. Access to it is provided by the Entrez system which can find identifiers (ID’s) and BLAST which can search sequence similarity. There are also FTP (bulk download) and programatic acess via the NCBI e-utilities system. The provided training materials for GenBank are very dense, and take some reading <a href="https://www.ncbi.nlm.nih.gov/guide/training-tutorials/">https://www.ncbi.nlm.nih.gov/guide/training-tutorials/</a>. The quick start guides are often a better place to start <a href="https://www.ncbi.nlm.nih.gov/nuccore/">https://www.ncbi.nlm.nih.gov/nuccore/</a></p>
</div>
<div id="geo" class="section level3" number="3.3.3">
<h3><span class="header-section-number">3.3.3</span> GEO</h3>
<p><a href="https://www.ncbi.nlm.nih.gov/geo/">GEO - https://www.ncbi.nlm.nih.gov/geo/</a></p>
<p>The Gene Expression Omnibus (GEO) is a functional genomics repository of microarray and RNAseq data that accepts raw microarray, sequence and curated gene expression profiles. The normal web, FTP and programmatic access routes are provided and help is provided best through the FAQ <a href="https://www.ncbi.nlm.nih.gov/geo/info/faq.html">https://www.ncbi.nlm.nih.gov/geo/info/faq.html</a>.</p>
</div>
<div id="biostudies" class="section level3" number="3.3.4">
<h3><span class="header-section-number">3.3.4</span> BioStudies</h3>
<p><a href="https://www.ebi.ac.uk/biostudies/about">BioStudies - https://www.ebi.ac.uk/biostudies/about</a></p>
<p>BioStudies is a relatively new database that consolidates some other databases. BioStudies database holds descriptions of biological studies and links to data from these studies in other databases. Here you can find transcriptome data (from ArrayExpress), literature data (from PubMedCentral), image data (from BioImages) and more. The help describes searching <a href="https://www.ebi.ac.uk/biostudies/help">https://www.ebi.ac.uk/biostudies/help</a> and again this database provides a web search interface, FTP and Aspera for bulk download and an API for programmatic access.</p>
</div>
<div id="protein-data-bank" class="section level3" number="3.3.5">
<h3><span class="header-section-number">3.3.5</span> Protein Data Bank</h3>
<p><a href="https://www.ebi.ac.uk/pdbe/node/1">Protein Data Bank https://www.ebi.ac.uk/pdbe/node/1</a></p>
<p>PDBe aims to provide a resource of high quality macromolecular structures with integration of function, taxonomy and sequence. The databank provides excellent tutorials at <a href="https://www.ebi.ac.uk/pdbe/training/tutorials">https://www.ebi.ac.uk/pdbe/training/tutorials</a> in particular the course at (<a href="https://www.ebi.ac.uk/training-beta/online/courses/biomacromolecular-structures" class="uri">https://www.ebi.ac.uk/training-beta/online/courses/biomacromolecular-structures</a>)[<a href="https://www.ebi.ac.uk/training-beta/online/courses/biomacromolecular-structures" class="uri">https://www.ebi.ac.uk/training-beta/online/courses/biomacromolecular-structures</a>] and the YouTube channel <a href="https://www.youtube.com/user/ProteinDataBank">https://www.youtube.com/user/ProteinDataBank</a></p>
</div>
</div>
<div id="secondary-databases" class="section level2" number="3.4">
<h2><span class="header-section-number">3.4</span> Secondary databases</h2>
<p>Secondary databases contain the results of analyses, often using the contents of primary databases. They will typically use information and raw data from various sources to create a new resource of information. Often highly curated, they can employ combinations of tools and analyses to present a synthesis of lower data types. These are the sort of databases molecular biologists are most used to using and many provide derived information of varying degrees of confidence about objects within the primary sequences, things like genes in genomes.</p>
</div>
<div id="examples-of-secondary-databases" class="section level2" number="3.5">
<h2><span class="header-section-number">3.5</span> Examples of secondary databases</h2>
<div id="ensembl" class="section level3" number="3.5.1">
<h3><span class="header-section-number">3.5.1</span> Ensembl</h3>
<p><a href="http://www.ensembl.org/index.html">http://www.ensembl.org/index.html</a></p>
<p>Ensembl is a genome browser based database that contains information on comparative genomics, sequence variation, transcriptional regulation alongside annotated genes. While <a href="ensembl.org">ensembl.org</a> is focussed on vertebrates, there are sister databases for fungi, plants and bacteria. The help files are large and have many video lessons <a href="http://www.ensembl.org/info/website/tutorials/index.html">http://www.ensembl.org/info/website/tutorials/index.html</a>. The BioMart tool in particular is very useful for medium scale retrieval <a href="http://www.ensembl.org/Multi/Help/Movie?db=core;id=189">BioMart</a>. The APIs provided can make whole genome and comparitive genomics analysis very streamlined.</p>
</div>
<div id="uniprotkb" class="section level3" number="3.5.2">
<h3><span class="header-section-number">3.5.2</span> UniProtKB</h3>
<p><a href="https://www.uniprot.org/help/uniprotkb">https://www.uniprot.org/help/uniprotkb</a></p>
<p>UniProt Knowledgebase is a database of functional information on proteins with a focus on accurate and consistent annotations. The database is divided into two parts, one with manually-curated records with information from literature and curator analyses (Swiss-Prot), the other with computationally-analysed only records that are yet to be examined by a human (TrEMBL). A YouTube channel of introductory videos is provided <a href="https://www.youtube.com/channel/UCkCR5RJZCZZoVTQzTYY92aw">https://www.youtube.com/channel/UCkCR5RJZCZZoVTQzTYY92aw</a> for high level information. A large selection of on-demand training is also available through EBI Training Portal, e.g <a href="https://www.ebi.ac.uk/training/events/guide-uniprot-students/">https://www.ebi.ac.uk/training/events/guide-uniprot-students/</a></p>
</div>
</div>
<div id="metadata" class="section level2" number="3.6">
<h2><span class="header-section-number">3.6</span> Metadata</h2>
<p>Data, especially raw data is absolutely useless on its own. It always need some contextual information to allow us to make use of it. This contextual data is called <code>metadata</code> - data about the data. For example genome sequencing data would need information about the organism sequenced, the machine used to sequence, the tissue type and much more. Anything that would be useful needs to be stored along with the data. Without metadata it would not possible to find data and select the correct samples for a given analysis of it.</p>
<p>In order for metadata to be useful we need to capture the same metadata for all data samples. This can mean not only standardising the questions we ask of, but also the words we allow to be used in answering the questions. This can seem a bit restrictive at first, but is actually essential. Consider two labs working on NLR proteins, one group is in the habit of calling them <code>NBS-LRR</code> proteins, the other <code>NLR</code>. These two terms would be completely different to a computer and anyone within the field not yet aware of all the jargon. Some searches and analyses and computer programs would completely fail to link these up.</p>
<p>As a result there are a wide range of accepted international standards on the metadata and this is why databases require users to annotate their data according to these standard. The standards themselves are constantly developing as new technologies and tools emerge, agreeing on data standards is an important and valuable job within research.</p>
<div id="minimum-information-standards" class="section level3" number="3.6.1">
<h3><span class="header-section-number">3.6.1</span> Minimum Information Standards</h3>
<p>Many data types have a ‘minimum information standard,’ a set of guidelines for reporting data on them. These ensure that the data of a certain type are all reported on in the same consistent way. Microarrays (MIAME), RNASeq (MINSEQE), and proteomics have data standards specific to them. More are listed at <a href="https://fairsharing.org/collection/MIBBI">FAIRSHARE</a>.</p>
</div>
<div id="controlled-vocabularies" class="section level3" number="3.6.2">
<h3><span class="header-section-number">3.6.2</span> Controlled Vocabularies</h3>
<p>To maintain a consistent set of terms for describing biological concepts and objects, we use controlled vocabularies, taxonomies and ontologies, among others. Controlled vocabularies restrict the words a human may use to describe something, a list of countries on a website would be an example of this. There are other ways to structure the controlled vocabulary. Taxonomies are ways of classifying things, often hierarchically from a more generic to a more specific group. This structure can add flexibility over the simple list, the biological taxonomy is an example of this. Ontologies represent the background knowledge of a domain, the objects are represented as terms and links between them are created to represent the relationships, such that we can analyse the ontology. The typical example is the Gene Ontology, which you can learn more about here <a href="https://www.ebi.ac.uk/training/online/course/goa-and-quickgo-quick-tour">https://www.ebi.ac.uk/training/online/course/goa-and-quickgo-quick-tour</a></p>
</div>
<div id="conclusion" class="section level3" number="3.6.3">
<h3><span class="header-section-number">3.6.3</span> Conclusion</h3>

<div class="roundup">
<pre><code>* Databases are important stores of data for the scientific record and for enabling new analyses. The metadata used to describe the data in databases is of paramount importance and is what makes the data useful.</code></pre>
</div>

</div>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="motivation.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="genomics-database-and-genome-assembly.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/rstudio/bookdown-demo/edit/master/02-primary-secondary.Rmd",
"text": "Edit"
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": null,
"toc": {
"collapse": "subsection"
}
});
});
</script>

</body>

</html>
